jena-arq/Grammar/master.jj

```diff
@@ -99,7 +99,42 @@ import org.apache.jena.sparql.core.Quad ;
 #endif
 
 public class CLASS extends PARSERBASE
-{}
+{
+    boolean allowAggregatesInExpressions = false ;
+#ifdef ARQ
```

Comment:
Please remove "main".

Reply:
Consider it done. I'll copy it to a GitHub gist in case I need to test grammars again (not only in Jena... I believe I actually got it from some StackOverflow answer).